site/data/docs/connect-button.mdx

```diff
@@ -74,6 +81,30 @@ Use the `label` prop to set a custom `ConnectButton` text.
 <ConnectButton label="__Sign in__" />
 ```
 
+#### onConnectChange
+
+Use the `onConnectChange` props to set callbacks executed when wallet connects or disconnects.
+
+When wallet connects.
+
+```tsx
+<ConnectButton
+  onConnectChange={({ isConnected, data }) => {
+    isConnected && console.log('gm!');
+  }}
+/>
+```
+
+When wallet disconnects.
+
+```tsx
+<ConnectButton
+  onConnectChange={({ isConnected, data }) => {
+    !isConnected && console.log('gn!');
+  }}
+/>
+```
+
```

Comment:
Should we add a note in this section that this callback will only get invoked after interacting with `ConnectButton`? Consumers could get the wrong assumption that the callback will be invoked for other cases such as: page refresh, users disconnecting the app from within their wallet, etc.

Reply:
Yes, that make sense

Reply:
SGMT!

Reply:
done!